halo/attest/types/tx.proto

```diff
@@ -54,11 +54,12 @@ message Vote {
 
 // BlockHeader uniquely identifies a cross chain block/attestation.
 message BlockHeader {
-  uint64 chain_id    = 1; // Chain ID as per https://chainlist.org
-  uint32 conf_level  = 2; // Confirmation level (aka version) of the cross-chain block/attestation.
-  uint64 offset      = 3; // Offset of the cross-chain block
-  uint64 height      = 4; // Height of the source-chain block
-  bytes  hash        = 5; // Hash of the source-chain block
+  uint64 source_chain_id      = 1; // Source Chain ID as per https://chainlist.org
+  uint64 consensus_chain_id   = 2; // Consensus Chain ID as per https://chainlist.org, for replay protection.
```

Comment:
I wouldn't add the chainlist.org bit here.

It won't be there. We don't get to reserve this chain ID cause it's not an evm chain. Someone else could reserve. And we wouldn't be able to support them

Reply:
I made this change in light of https://github.com/omni-network/ops/issues/198, but it's not super relevant to this PR anyway. I'll simplify for now regardless.

Reply:
I mean the "Consensus chain as per http://chainlist.org"

Reply:
Yeah, I believe we're planning on registering the consensus chain ID with chainlist too ([thread](https://omni-chain.slack.com/archives/C02HCJN6KNE/p1721318167863919?thread_ts=1721249907.530149&cid=C02HCJN6KNE)), but regardless dropped from this PR.